src/main/java/de/tum/in/www1/artemis/service/LectureService.java

```diff
@@ -38,11 +43,17 @@ public class LectureService {
 
     private final ChannelService channelService;
 
-    public LectureService(LectureRepository lectureRepository, AuthorizationCheckService authCheckService, ChannelRepository channelRepository, ChannelService channelService) {
+    private final Optional<PyrisWebhookService> pyrisWebhookService;
+
+    private static final Logger log = LoggerFactory.getLogger(LoggingAspect.class);
```

Comment:
Most of the time the logger is the very first attribute. Can you do it here as well?